testing/py-parso/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Kevin Daudt <kdaudt@alpinelinux.org>
+# Maintainer: Kevin Daudt <kdaudt@alpinelinux.org>
+pkgname=py-parso
+pkgver=0.3.4
+pkgrel=0
+pkgdesc="a Python parser that supports error recovery and round-trip parsing for different Python versions"
+url="https://pypi.org/project/parso/"
+arch="noarch"
+license="MIT"
+makedepends="python3-dev py3-setuptools"
+checkdepends="py3-tox"
+depends="python3"
+source="https://files.pythonhosted.org/packages/source/p/parso/parso-$pkgver.tar.gz"
+builddir="$srcdir/parso-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	tox -e py36 # This should match the current python version
```

Comment:
You could try something like so it will work on upcoming 3.7 upgrade.
```
local _pyver=$(python3 -c 'import sys; print(str(sys.version_info[0])+str(sys.version_info[1]))')
tox -e py$_pyver
```

Reply:
Thanks, was indeed thinking about how to solve that.